volmdlr/edges.py

```diff
@@ -1621,26 +1624,48 @@ def point_belongs(self, point2d, abs_tol=1e-10):
         """
         check if a Point2D belongs to the Arc2D
         """
-        vector_start = self.start - self.center
-        vector_end = self.end - self.center
-        vector_point = point2d - self.center
-        r1 = vector_start.norm()
-        cp = vector_point.norm()
-        if math.isclose(cp, r1, abs_tol=abs_tol):
-            if (self.start.x < self.end.x) and (self.start.y < self.end.y):
-                arc_angle = - volmdlr.core.clockwise_angle(vector_start,
-                                                           vector_end)
-                point_angle = - volmdlr.core.clockwise_angle(vector_start,
+
+        if (self.start.point_distance(point2d) <= abs_tol
+                or self.end.point_distance(point2d) <= abs_tol):
+            return True
+
+        if self.is_trigo:
+            arc_trigo = self.reverse()
+        else:
+            arc_trigo = self
```

Comment:
```suggestion
        arc_trigo = self
        if self.is_trigo:
            arc_trigo = self.reverse()
           
```

Reply:
If you really want to make this compact:
```suggestion
        arc_trigo = self.reverse() if self.is_trigo else self
```

Reply:
Done !